specs/DesignDiscussions/InputLabels.md

```diff
@@ -91,6 +129,42 @@ If we want to allow the host to control formatting of the labels, either in gene
 ```
 Another option is to not put this in host config, and instead control this styling via support of native styling on required and optional labels (i.e. CSS, Xaml Styles, etc). This decision speaks to a larger discussion of the direction of host config that is beyond the scope of this document.
 
+> ### Host side styling
+> 
+> Following the previously mentioned text formatting the proposed set of properties we could allow the host application to override would be almost the same set of properties we provide for configuring a TextBlock: 
+> 
+> | Property | Default Value |HostConfig | Native |
+> | --- | --- | --- | --- |
+> | spacing | "default" | X | |
+> | fontType | "default" | X | X |
+> | size | "default" | X | X |
+> | weight | "default" | X | X |
+> | color | "default" | X | X |
+> | isSubtle | false | X | |
+> | separator | false | X | |
+>
+> ```json
+> "inputLabelFormatting": 
+> {
+>	"requiredInputs": {
+>		"spacing": "large",
+>		"weight": "Bolder",
+>		"fontType": "monospace",
+>		"size": "medium",
+>		"color": "good"
+>	},
+>	"optionalInputs": {
+>		"isSubtle": true
+>	}	
+> }
+> ```
+>
+> While it's mentioned that we should allow almost all TextBlock properties to be modified by the label property, some of them must stay out of bounds for hosts as they would provide a bad user experience:
+> * maxLines and wrap: Labels should be descriptive enough so users can provide the information needed, if the user can't read the whole question the information they will not be able to provide a correct answer
+> * isVisible: labels should always be visible as long as the associated input is
+> * height: input elements should control the height for the whole set of label, input and error message 
+>
+
 #### Required and Optional indicators
 
  By default, we will mark required inputs with a `*`. We may want, however, to provide the option for the host to configure a suffix to the label for required or optional inputs. This allows the host to add a `*` to required labels, or to add the words "required" or "optional". Consider a host config similar to the below:
```

Comment:
>with a `*`. [](start = 42, length = 11)

Great! When we say by default, that means the default host config(urable) value will be "*" for suffix property for requiredinputs right?  And are we proposing we have any defaults for the optionalInputs defined "suffix" ?